formatAmount/index.ts

```diff
@@ -4,7 +4,12 @@ export default (amount: number, options: { decimals?: boolean; alwaysNegative?:
     amount = -Math.abs(amount)
   }
 
-  return amount.toLocaleString('nb-NO', {
+  /*17.08.21: Bug in chrome making norwegian locale not available. 
+  Adding polyfills broke login, so let's temporarily (tm) use swedish locale here, which is not broken.
+  https://gitmemory.com/issue/formatjs/formatjs/3066/888110768
+  */
+
+  return amount.toLocaleString('sv-SE', {
```

Comment:
Hmmmm, check for Chrome version? I fell this could cause issues in other browsers...

Reply:
Or - pick first one matching from sv-SE, nb-NO (pl-PL)?

Reply:
window.Intl.NumberFormat.supportedLocalesOf(['nb-NO', 'nn-NO', 'sv-SE', 'pl-PL', 'en-US'])[0]
(not sure if this works in node tho)?

Reply:
Or we can wait until https://bugs.chromium.org/p/chromium/issues/detail?id=1215606 is rolled out :)